extensions/sql-bindings/src/test/service/azureFunctionsService.test.ts

```diff
@@ -0,0 +1,201 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as should from 'should';
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as utils from '../../common/utils';
+import * as constants from '../../common/constants';
+import * as azureFunctionUtils from '../../common/azureFunctionsUtils';
+import * as azureFunctionsContracts from '../../contracts/azureFunctions/azureFunctionsContracts';
+import * as azureFunctionService from '../../services/azureFunctionsService';
+
+import { createTestUtils, TestUtils, createTestCredentials, createTestTableNode } from '../testUtils';
+import { IConnectionInfo } from 'vscode-mssql';
+import { BindingType } from 'sql-bindings';
+
+const rootFolderPath = 'test';
+const projectFilePath: string = path.join(rootFolderPath, 'test.csproj');
+let testUtils: TestUtils;
+describe('Tests to verify Create Azure Function with SQL Binding', () => {
```

Comment:
nit: But we know these are tests (they're in a test file and will only show up when running tests...) so I prefer to just keep these as simple as direct as possible. 

Describe/Suite should just be the "area" of the tests. Typically just 1-3 words and then nested as appriate. So this file would have a top level `AzureFunctionsService` describe, with one under it for `createAzureFunction`. And then within that would go each separate test for that function. 


```
describe('AzureFunctionsService', () => {
   describe('createAzureFunction', () => {
        it(...)
        ...
```

Laying it out like this makes it really easy to figure out where new tests should go and to find existing tests.